d/dotnet7/dotnet7-ubuntu18.04.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package   : dotnet
+# Version   : 7.0
+# Source repo   : 
```

Comment:
@Sapana-Khemkar,
Just a suggestion,
I am sure you are dealing with multiple repos here.
However, can you mention one github URL(probably, runtime github URL) here ?
`githuburl is kind of an mandatory requirement for automation.`

CC: @seth-priya Correct me if I am wrong.

Reply:
@kandarpamalipeddi changes done.